cmd/queue/main.go

```diff
@@ -118,25 +123,29 @@ var (
-	ContainerConcurrency         int    `split_words:"true" required:"true"`
-	QueueServingPort             int    `split_words:"true" required:"true"`
-	RevisionTimeoutSeconds       int    `split_words:"true" required:"true"`
-	UserPort                     int    `split_words:"true" required:"true"`
-	EnableVarLogCollection       bool   `split_words:"true"` // optional
-	ServingConfiguration         string `split_words:"true" required:"true"`
-	ServingNamespace             string `split_words:"true" required:"true"`
-	ServingPodIP                 string `split_words:"true" required:"true"`
-	ServingPod                   string `split_words:"true" required:"true"`
-	ServingRevision              string `split_words:"true" required:"true"`
-	ServingService               string `split_words:"true"` // optional
-	UserContainerName            string `split_words:"true" required:"true"`
-	VarLogVolumeName             string `split_words:"true" required:"true"`
-	InternalVolumePath           string `split_words:"true" required:"true"`
-	ServingLoggingConfig         string `split_words:"true" required:"true"`
-	ServingLoggingLevel          string `split_words:"true" required:"true"`
-	ServingRequestMetricsBackend string `split_words:"true" required:"true"`
-	ServingRequestLogTemplate    string `split_words:"true" required:"true"`
-	ServingReadinessProbe        string `split_words:"true" required:"true"`
+	ContainerConcurrency         int     `split_words:"true" required:"true"`
+	QueueServingPort             int     `split_words:"true" required:"true"`
+	RevisionTimeoutSeconds       int     `split_words:"true" required:"true"`
+	UserPort                     int     `split_words:"true" required:"true"`
+	EnableVarLogCollection       bool    `split_words:"true"` // optional
+	ServingConfiguration         string  `split_words:"true" required:"true"`
+	ServingNamespace             string  `split_words:"true" required:"true"`
+	ServingPodIP                 string  `split_words:"true" required:"true"`
+	ServingPod                   string  `split_words:"true" required:"true"`
+	ServingRevision              string  `split_words:"true" required:"true"`
+	ServingService               string  `split_words:"true"` // optional
+	UserContainerName            string  `split_words:"true" required:"true"`
+	VarLogVolumeName             string  `split_words:"true" required:"true"`
+	InternalVolumePath           string  `split_words:"true" required:"true"`
+	ServingLoggingConfig         string  `split_words:"true" required:"true"`
+	ServingLoggingLevel          string  `split_words:"true" required:"true"`
+	ServingRequestMetricsBackend string  `split_words:"true" required:"true"`
+	ServingRequestLogTemplate    string  `split_words:"true" required:"true"`
+	ServingReadinessProbe        string  `split_words:"true" required:"true"`
+	TracingConfigDebug           bool    `split_words:"true"` // optional
+	TracingConfigEnable          bool    `split_words:"true" required:"true"`
```

Comment:
is it really required?

Reply:
If it's not set, it will just default to false. We can remove required.